src/test/java/HtmlUnitDriverRMSysLoginTest.java

```diff
@@ -0,0 +1,20 @@
+import core.TestBaseHtmlUnit;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HtmlUnitHomePage;
+import pages.HtmlUnitLoginPage;
+
+public class HtmlUnitDriverRMSysLoginTest extends TestBaseHtmlUnit {
+    public static final String USER_NAME = "EugenBorisik";
+    public static final String PASSWORD = "qwerty12345";
```

Comment:
Please. make private. I recommend to make all fields private first and only then if needed change to protected/public. It will help you to avoid such mistakes in future.